gazebo_ros_control/CMakeLists.txt

```diff
@@ -31,9 +33,11 @@ catkin_package(
     joint_limits_interface
     urdf
     angles
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME} default_robot_hw_sim
-  DEPENDS gazebo
```

Comment:
> It just says DEPENDS - Non-catkin CMake projects that this project depends on but then why does it also not include things like Boost?
> 
> Another example - why does gazebo_ros not DEPEND on GAZEBO?

I would say that both are bugs or errors in the configuration of our CMakeLists.txt. I found this [answer](http://answers.ros.org/question/58498/what-is-the-purpose-of-catkin_depends/) specially useful to understand what are they used for. Now it makes sense that changes in `DEPEND` will affect third party software.